### PR TITLE
Openvino benchmarking

### PR DESCRIPTION
**Pull request by ynimmaga**
OpenVINO is Intel's inferencing toolkit to accelerate performance on Intel CPUs, integrated and discrete GPUs. It is added as a torch.compile backend. More information can be found here: https://pytorch.org/docs/stable/torch.compiler.html. This PR enables users to use openvino backend for torchdynamo benchmarking.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang

### CODE REVIEW

**Comment by colesbury** [comment]
@jansel, would you please review this?

File: benchmarks/dynamo/common.py [hunk 1]
```diff
@@ -95,6 +95,18 @@
 
 log = logging.getLogger(__name__)
 
+# Enable OpenVINO Backend for benchmarking
+try:
+    import openvino.torch
```
Comment (jansel) [inline benchmarks/dynamo/common.py:100]: You shoudn't actually need to import a backend for it to work with `torch.compile(backend="openvino")` if you configure your package properly.  This is also true for benchmarking with `--backend=openvino`

This PR description explains how to setup an entrypoint for your package: https://github.com/pytorch/pytorch/pull/93873

This code will import your package:
https://github.com/pytorch/pytorch/blob/ea61c9cb299b6dfebc57dc9d8821c34321d568ab/torch/_dynamo/backends/registry.py#L98-L115

**Review by jansel — CHANGES_REQUESTED** [review]

**Comment by ynimmaga** [comment]
Will add the suggested change in the openvino package.

File: benchmarks/dynamo/common.py (hunk 1 above)
Comment (ilya-lavrenov) [reply benchmarks/dynamo/common.py:100]: how does it work with conda-forge where we cannot (?) specify torch entry points?

**Pull request closed without merging**